Cell.java

```diff
@@ -0,0 +1,57 @@
+public class Cell 
+{
+    Board parent; // parent board
+    boolean [] Posibilities; // if the cell is empty, what numbers could be in that cell
+    boolean isPlaced; // is this cell empty?
+    int num; // the number the cell contains
+    
+    // coordinates
+    int x;
+    int y;
+    
+    // constructor, empty Cell
+    public Cell(Board p, int x, int y){
+        
+        num = 0; 
+        Posibilities = new boolean[] {true,true,true,true,true,true,true,true,true};

```

Comment:
Generate a true * x somehow, this is ugly

Reply:
done